app/controllers/users_controller.rb

```diff
@@ -30,10 +30,11 @@ def create
     set_user_fields
 
     if @user.errors.empty? && @user.save
+      send_email_confirmation_mail(@user)
       if @bare_layout
-        render text: '<div class="success" style="font-size: 14pt; margin: 10pt;">User account created.</div>', layout: true
+        render text: '<div class="success" style="font-size: 14pt; margin: 10pt;">User account created. Confirmation email has been sent to your email address.</div>', layout: true
       else
-        flash[:notice] = 'User account created. You can now log in.'
+        flash[:notice] = 'User account created. Confirmation email has been sent to your email address.'
```

Comment:
We probably should throw in the mention to check the spam folder should they have problems receiving the email